azurerm/resource_arm_automation_schedule_test.go

```diff
@@ -0,0 +1,126 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/arm/automation"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMAutomationSchedule_oneTime(t *testing.T) {
+	ri := acctest.RandInt()
+	config := testAccAzureRMAutomationSchedule_oneTime(ri, testLocation())
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMAutomationScheduleDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMAutomationScheduleExistsAndFrequencyType("azurerm_automation_schedule.test", automation.OneTime),
```

Comment:
(as above) - can we update this to be `testCheckAzureRMAutomationScheduleExists` and check the values in the state?

Reply:
Done